src/System.Linq/src/System/Linq/In.cs

```diff
@@ -0,0 +1,38 @@
+﻿//NiceLinq by Mohamed Elshawaf m_shawaf@outlook.com
+using System.Collections.Generic;
+using System.Diagnostics;
+using static System.Linq.Utilities;
+
+namespace System.Linq
+{
+    public static partial class Enumerable
```

Comment:
Enumerable and Queryable should correspond with each other as much as possible, so it shouldn't be done there *instead of* but *as well as*. It will still need to start with an issue proposing the change because it changes the public API (and I'm not sure it offers very much, but then maybe others will disagree with me on that).